tests/e2e/tests/configure-and-run-analysis.test.ts

```diff
@@ -35,10 +35,131 @@ test.describe(`Configure extension and run analysis`, () => {
     await vscodeApp.runAnalysis();
     await vscodeApp.waitDefault();
     await expect(vscodeApp.getWindow().getByText('Analysis completed').first()).toBeVisible({
-      timeout: 300000,
+      timeout: 400000,
     });
   });
 
+  const collator = new Intl.Collator(undefined, { sensitivity: 'base', numeric: true });
+  const isAscending = (arr: string[]) =>
+    arr.every((v, i, a) => i === 0 || collator.compare(a[i - 1], v) <= 0);
+
+  /**
+   * Checks that the issues list is sorted correctly in both ascending and descending order.
+   * Verifies that the descending order is not ascending, and ascending order is sorted.
+   */
+  test('Set list kind and sort (Issues ascending and descending)', async () => {
+    await vscodeApp.setListKindAndSort('issues', 'descending');
+    const names_descending = await vscodeApp.getListNames('issues');
+    //console.log('Issue names:', names_descending);
+    expect(isAscending(names_descending)).toBe(false);
+
+    await vscodeApp.setListKindAndSort('issues', 'ascending');
+    const names_ascending = await vscodeApp.getListNames('issues');
+    //console.log('Issue names:', names_ascending);
+    expect(isAscending(names_ascending)).toBe(true);
+  });
+
+  /**
+   * Checks that each file name in the provided list is valid.
+   * A valid file name has a known bare name or a file extension.
+   * @param names - The list of file names to validate.
+   */
+  function expectFilesListLooksValid(names: string[]) {
+    const knownBare = new Set([
+      'README',
+      'LICENSE',
+      'Makefile',
+      'Dockerfile',
+      'BUILD',
+      'WORKSPACE',
+    ]);
```

Comment:
I don't think this is the way to go, there is an big list of valid files that could live in the root of a java project, and an unlimite one if you create custom files.

For example in Gradle projects we may have gradlew. And in any project we may have things like "CONTRIBUTING", "NOTICE", etc.